### ISSUE TITLE
Undefined name 'PolygonUtil'. 

### ISSUE THREAD

**Issue by fredxie07** [kb0/maps_toolkit#3]
I got the following error :
Try correcting the name to one that is defined, or defining the name.

**Comment by yuanchen1** [comment]
![image](https://user-images.githubusercontent.com/35587570/84988877-55faa100-b175-11ea-8cc0-58287b5c1c3b.png)

Why didn't  export it

**Comment by fredxie07** [comment]
```
/// Maps toolkit (port from android-maps-utils library)
library maps_toolkit;

export 'src/latlng.dart';
export 'src/spherical_util.dart';
```

We found the reason ,because  didn't export this tool

**Comment by kb0** [comment]
fixed, at 1.1.0+1
